src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -12,7 +26,171 @@ public SEDTweaksSteps()
+	    String a5 = driver.findElement(By.cssSelector("div.panel:nth-child(6) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a5);
+	    String a6 = driver.findElement(By.cssSelector("div.panel:nth-child(7) > div:nth-child(1) > a:nth-child(1)")).getText();
+	    System.out.println("Accordion title:" +a6);
+	    
+	    
+	}
+
+	@Then("^I should see them in the following order: Criteria, CQMs, SED, G(\\d+)/G(\\d+), Surveillance, Additional Info$")
+	public void i_should_see_them_in_the_following_order_Criteria_CQMs_SED_G_G_Surveillance_Additional_Info(int arg1, int arg2) throws Throwable {
+		
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Certification Criteria"));
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(3) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Clinical Quality Measures"));
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Safety Enhanced Design (SED)"));
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(5) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Successfully Tested G1/G2 Measures"));
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(6) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Surveillance Activities"));
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(7) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Additional Information"));
+		
+	}
+
+	@Given("^I am looking at a listing's SED details$")
+	public void i_am_looking_at_a_listing_s_SED_details() throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/product/8481");
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")));
+		
+	}
+
+	@When("^I look at the SED Intended User Description$")
+	public void i_look_at_the_SED_INtended_User_Description() throws Throwable {
+		
+	
+		Thread.sleep(5000);
+		WebElement expanded = driver.findElement(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)"));
+		expanded.click();
+					
+		WebElement expanded1 = driver.findElement(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)"));
+		expanded1.click();
+		
+		if(driver.getPageSource().contains("No Certification Criteria were tested for SED."))
+		    System.out.println("No Certification Criteria were tested for SED.");
+		else
+			driver.getPageSource().contains("Description of Intended Users");
+	}
+
+	@Then("^the header should be named Description of Intended Users$")
+	public void the_header_should_be_named_Description_of_Intended_Users() throws Throwable {
+	    
+		assertTrue(
+				driver.getPageSource().contains("Description of Intended Users"));
```

Comment:
Will this be true if the header is anywhere on the page? Should it be adjusted to find a particular thing, whether by class, id, or page structure?

Reply:
If this is succeeding for now, I'm not sure if it needs to be fixed for the PR